src/cloud-api-adaptor/hack/build.sh

```diff
@@ -59,10 +59,13 @@ function build_caa_payload_image() {
 		--build-arg COMMIT="${commit}" \
 		--build-arg YQ_VERSION="${YQ_VERSION}" \
 		--build-arg YQ_CHECKSUM="${YQ_CHECKSUM}" \
+		--load \
 		-f cloud-api-adaptor/Dockerfile \
 		${tag_string} \
-		--push \
 		.
+
+    docker push  ${registry}/${name}
```

Comment:
@davidhadas would you like fix the indent here, maybe replace the `blankspaces` with `TAB`?